layouts/_default/baseof.html

```diff
@@ -15,7 +15,7 @@
       <div>
         <div class="site-title">
           <a href="/">
-            <img src="/images/WebFinger.svg" id="logo" alt=""/>
+            <img src="/images/WebFinger.svg" id="logo" alt="" width="293" height="452"/>
```

Comment:
I think this should be the dimensions of the space the image takes on the page, not necessarily the original dimensions of the image.  So this should probably be `width="49" height="75"`.

Reply:
Makes sense! Just updated the PR with the changes.